### ISSUE TITLE
Error building che-plugin-registry

### ISSUE THREAD

**Issue by ghost** [eclipse/che#21583]
### Describe the bug

I try to build che-plugin-registry according to the README instructions and it fails with the following output:
[out.txt](https://github.com/eclipse/che/files/9183055/out.txt)

### Che version

7.49

### Steps to reproduce

1. Clone https://github.com/eclipse/che-plugin-registry
2. Run build.sh
3. Error

### Expected behavior

It should build.

### Runtime

other (please specify in additional context)

### Installation method

other (please specify in additional context)

### Environment

Linux

### Additional context

I selected version 7.49.0 because that's the version of che/chectl I'm trying to use, but this happens to me also on 7.50.0 and on master.

Environment: Fedora 36, npm 8.3.1, yarn 3.0.2, node 16.14.0

**Comment by ibuziuk** [comment]
Could you please try against the latest 7.51.0 cc: @svor

**Comment by yujiri8** [comment]
7.51.0 Failed with a similar error, though a different plugin:

[out.txt](https://github.com/eclipse/che/files/9184526/out.txt)

Information I should've included earlier: With the other versions I tried, the path `/tmp/che-plugin-registry/unpack-folder/` was completely empty (maybe it got removed after the error? I don't know). With 7.51.0, a few packages are in there, including the one cited in the error, vscode-tekton-pipelines-0.2.0, and it has an `extension` folder in it, but that folder is empty.

With the other packages in there, such as xtend-lang-0.1.0, `extension/package.json` exists.

**Comment by svor** [comment]
@yujiri8 try to remove **rm -rf /tmp/che-plugin-registry** and restart build.
I can't reproduce this problem on my local machine and as I see GitHub action doesn't have such problem with the build: https://github.com/eclipse-che/che-plugin-registry/actions/workflows/next-build.yml

**Comment by yujiri8** [comment]
Strange... that indeed worked. Perhaps the intermittent 'failed to start axios' error got it into an innvalid state in the first place.

**Comment by yujiri8** [comment]
So, I got this to build 2 days ago, now can't. I'm not aware of anything that changed. There are 2 failure modes I'm seeing:
1. It just gets stuck forever after downloading some of the plugins. I've seen it get to 71/72 and then waited more than 10 minutes without it progressing past that.
2. It crashes with a connection refused error while downloading one of the plugins from github: see this log:
[out.txt](https://github.com/eclipse/che/files/9214831/out.txt)

I'd seen failure #2 before, so I was absolutely convinced it was an intermittent failure, and so I've been trying the process again and again all day until finally I came to the reality that I'm not going to see a successful build again. Something must have changed.

I'm always able to reach the IP address indicated, so I don't think it's actually a problem with my network connection. It is a Github Pages IP address.

Any clues?
